docs/INSTALLATION.md

```diff
@@ -11,7 +11,79 @@ of Perl 5, take a look at 'Other Options'.
 If you are using Fedora/Red Hat/CentOS, some core modules are not included with Perl.
 Use the `yum install perl-core` command to install them.
 
-### Other Options:
+---
+
+### Install Basic Development Tools
+
+#### RedHat, CentOS, Fedora and similar
+
+Use the following set of commands to install development packages on your RedHat and their derivative systems.
+
+```shell
+
+### CentOS/RHEL 7/6/5 
+sudo yum update
+sudo yum groupinstall "Development Tools"
+
+### Fedora 28/27/26/25/24/23 
+sudo dnf update
+sudo dnf groupinstall "Development Tools"
+
+```
+
+#### Debian, Ubuntu and similar
+
+```shell
+
+sudo apt update
+sudo apt install build-essential
+
+```
+
+#### Mac OSX
+
+```shell
+
+xcode-select --install
+
+```
+
+---
+
+### Install Test Suite
+The Perl 5 track in Exercism uses [Test2 Suite](https://metacpan.org/pod/Test2::Suite) 
+to verify the codes. It's mandatory installing this suite to have your tests verified.
+
+The last bundle is _Test2::V0_.
+
+We strongly recommend you installing local::lib and cpanminus, 
+so you could install packages and work under your own workspace. 
+
+```shell
+
```

Comment:
Extra blank lines around the commands to invoke.

Reply:
Done. All unnecessary blank lines were removed. Thank you, Kotp for your attention! :+1: